scripts/installers/docker-setup.sh

```diff
@@ -69,7 +69,7 @@ function pull_image() {
 function setup_mountpoints() {
     echo -e "${RED}Creating mount points${NC}"
     for dev in /dev/sr?; do
-        sudo -u arm mkdir -p "/mnt$dev"
+        sudo mkdir -p "/mnt$dev"
     done
 }
```

Comment:
Insert `chown arm:arm /mnt/dev/sr*` on this line. This goes after the loop so that we can make `arm` the owner of all of the newly created folders in a single command